lib/Sharing/DeckShareProvider.php

```diff
@@ -75,16 +76,29 @@ class DeckShareProvider implements \OCP\Share\IShareProvider {
 	private $cardMapper;
 	/** @var PermissionService */
 	private $permissionService;
+	/** @var AttachmentService */
+	private $attachmentService;
 	/** @var ITimeFactory */
 	private $timeFactory;
 	private $l;
 
-	public function __construct(IDBConnection $connection, IManager $shareManager, ISecureRandom $secureRandom, BoardMapper $boardMapper, CardMapper $cardMapper, PermissionService $permissionService, IL10N $l) {
+	public function __construct(
+		IDBConnection $connection, 
+		IManager $shareManager, 
+		ISecureRandom $secureRandom, 
+		BoardMapper $boardMapper, 
+		CardMapper $cardMapper, 
+		AttachmentService $attachmentService,
+		PermissionService $permissionService, 
+		IL10N $l
+	) {
 		$this->dbConnection = $connection;
 		$this->shareManager = $shareManager;
 		$this->boardMapper = $boardMapper;
 		$this->cardMapper = $cardMapper;
+		$this->attachmentService = $attachmentService;
```

Comment:
Injecting this service seems to cause a cyclic dependency and therefore crashing the php process on the test runs